Source/BoundaryConditions/WarpXEvolvePML.cpp

```diff
@@ -301,27 +304,21 @@ WarpX::DampJPML (int lev, PatchType patch_type)
 
             amrex::ParallelFor( tjx, tjy, tjz,
                 [=] AMREX_GPU_DEVICE (int i, int j, int k) {
-#ifdef AMREX_USE_EB
-                    if(pml_lxfab(i, j, k) <= 0) return;
-#endif
+                    if (pml_lxfab && pml_lxfab(i, j, k) <= 0) { return; }
```

Comment:
Could we rename `pml_l[xyz]fab` as `eb_l[xyz]fab`?
This is because, with the `ifdef` not here anymore, it may not be clear to someone reading the code that this is related to embedded boundaries.
(We could also add a comment: `// Skip the field update if this gridpoint is inside the embedded boundary`.)